nextest-runner/src/cargo_config/target_triple.rs

```diff
@@ -24,6 +24,18 @@ pub struct TargetTriple {
 }
 
 impl TargetTriple {
+    /// Create an x86_64-unknown-linux-gnu [`TargetTriple`]. Useful for testing.
+    ///
+    /// # Panics
+    ///
+    /// Panics if the underlying implementation fail to parse the `"x86_64-unknown-linux-gnu"`
+    /// triple string.
+    pub fn x86_64_unknown_linux_gnu() -> Self {
```

Comment:
Can this be `cfg(test)`?

Reply:
Yes, it can.

Reply:
It can -- addressing this and other followups in comments.